testkit/mockstore.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !codes
+
+package testkit
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/session"
+	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateMockStore return a new mock kv.Storage.
+func CreateMockStore(t *testing.T) (store kv.Storage, clean func()) {
+	store, err := mockstore.NewMockStore()
+	require.NoError(t, err)
+
+	session.SetSchemaLease(0)
+	session.DisableStats4Test()
+	d, err := session.BootstrapSession(store)
```

Comment:
I have a concern this will make our CI take a lot of time than before.
In the past, one test suite create one store/domain.
If after the refactor, each test case create one store/domain, the operation is heavier than before... `session.BootstrapSession` is not cheap.

Reply:
OK. I'll think of reusing store/domain between tests. Maybe we come back to batch several tests in serial with testify suite.